infrastructure/src/main/java/org/corfudb/infrastructure/log/FileSystemAgent.java

```diff
@@ -129,4 +157,46 @@ private void checkLimits() {
             }
         }
     }
+
+    public static class PartitionAgent {
+        @Getter
+        private PartitionAttribute partitionAttribute;
+        private final Path logPartition;
```

Comment:
This does not seem to be used anywhere

Reply:
Chetan has been implemented that functionality https://github.com/CorfuDB/CorfuDB/pull/3121